src/windows/editor_object_selection.c

```diff
@@ -56,6 +56,9 @@ enum {
 	FILTER_ALL = 0x7EF,
 } FILTER_FLAGS;
 
+bool _filter_selected = false;
+bool _filter_nonselected = false;
```

Comment:
Do you mean add them into the filter flags enum right above this spot? if so, should i place them between the "FILTER_CUSTOM = (1 << 3)," and "FILTER_RIDE_TRANSPORT = (1 << 5)," (with skipping a number before and after the selected/nonselected)? or should i place them at the end of the enum that way incase theres some references to the ride type being required to be the certain numbers they currently are, they won't be disturbed?